install on mac client fail

-- ARCHITECTURE: x86_64
-- BUILD_DYNAMIC_LIB:  ON
-- BUILD_STATIC_LIB:  ON
-- BUILD_TESTS:  ON
-- BUILD_PYTHON_WRAPPER:  ON
-- BUILD_PERF_TOOLS:  OFF
-- LINK_STATIC:  OFF
-- USE_LOG4CXX:  OFF
-- CMAKE_BUILD_TYPE:  RelWithDebInfo
-- Threads library: 
Failed to find Protobuf in config mode, try to find it from system path
-- Protobuf_LIBRARIES: /usr/local/lib/libprotobuf.dylib
-- Protobuf_INCLUDE_DIRS: /usr/local/include
-- Found Boost: /usr/local/include (found version "1.76.0")  
-- Using std::regex
-- PYTHON: 3.7.7
-- DETECTED Python 3
-- Found Boost: /usr/local/include (found version "1.76.0") found components: python39 
-- HAS_ZSTD: 1
-- HAS_SNAPPY: 0
-- Using Boost Python libs: /usr/local/lib/libboost_python39-mt.dylib;Boost_PYTHON37-MT_LIBRARY_RELEASE-NOTFOUND
CMake Error at python/CMakeLists.txt:85 (MESSAGE):
  Could not find Boost Python library


-- Configuring incomplete, errors occurred!
See also "/Users/admin/Documents/study/workspace/trade-test/third_party/pulsar/pulsar-client-cpp/CMakeFiles/CMakeOutput.log".

--- a/pulsar-client-cpp/python/CMakeLists.txt
+++ b/pulsar-client-cpp/python/CMakeLists.txt
@@ -72,11 +72,18 @@ set(PYTHON_WRAPPER_LIBS ${Boost_PYTHON_LIBRARY}
                         ${Boost_PYTHON39_LIBRARY})
 
 if (APPLE)
-    set(PYTHON_WRAPPER_LIBS ${PYTHON_WRAPPER_LIBS}
-                            ${Boost_PYTHON27-MT_LIBRARY_RELEASE}
-                            ${Boost_PYTHON37-MT_LIBRARY_RELEASE}
-                            ${Boost_PYTHON38-MT_LIBRARY_RELEASE}
-                            ${Boost_PYTHON39-MT_LIBRARY_RELEASE})
+    if (Boost_PYTHON27-MT_LIBRARY_RELEASE)
+        set(PYTHON_WRAPPER_LIBS ${PYTHON_WRAPPER_LIBS} ${Boost_PYTHON27-MT_LIBRARY_RELEASE})
+    endif ()
+    if (Boost_PYTHON37-MT_LIBRARY_RELEASE)
+        set(PYTHON_WRAPPER_LIBS ${PYTHON_WRAPPER_LIBS} ${Boost_PYTHON37-MT_LIBRARY_RELEASE})
+    endif ()
+    if (Boost_PYTHON38-MT_LIBRARY_RELEASE)
+        set(PYTHON_WRAPPER_LIBS ${PYTHON_WRAPPER_LIBS} ${Boost_PYTHON38-MT_LIBRARY_RELEASE})
+    endif ()
+    if (Boost_PYTHON39-MT_LIBRARY_RELEASE)
+        set(PYTHON_WRAPPER_LIBS ${PYTHON_WRAPPER_LIBS} ${Boost_PYTHON39-MT_LIBRARY_RELEASE})
+    endif ()
 endif()
 
 message(STATUS "Using Boost Python libs: ${PYTHON_WRAPPER_LIBS}")

